### DISCUSSION TITLE
How to use the mixer with Focusrite Scarlett 4i4 4th Gen

### DISCUSSION THREAD

**Discussion by bulkware** [geoffreybennett/alsa-scarlett-gui#97] [Q&A] [answered]
Hello fellow Linux enthusiasts! I am trying to figure out how to use the mixer. Can someone explain to me what is what and how to use it? Compare this to the Windows application, this is quite a jungle of controls :)

![focusrite-linux-mixer](https://github.com/geoffreybennett/alsa-scarlett-gui/assets/3463048/702263d8-5297-401d-a750-9b4757b9e57b)
![focusrite-windows-mixer](https://github.com/geoffreybennett/alsa-scarlett-gui/assets/3463048/a43fa6ab-db22-4c38-b5d0-eaef72d57d31)
![focusrite-windows-inputs](https://github.com/geoffreybennett/alsa-scarlett-gui/assets/3463048/4e135e25-5c10-43c0-9c7a-622d1c05e7e1)

**Comment by geoffreybennett** [comment, marked as answer]
Hi @bulkware,

alsa-scarlett-gui isn't quite ready for public consumption yet. You will need to build it yourself from the dev branch to access all the features.

It's more complicated than Focusrite Control for a few reasons. Mainly that it's currently a direct reflection of the hardware controls:
- There's no way to indicate that certain channels should be treated as mono rather than stereo
- The DSP channels are exposed rather than hidden
- The PCM (USB) Input channels can all be arbitrarily assigned

I'm still working on making it more user-friendly.

To understand the mixer window, you need to refer to the routing window, to see what Mix A–F are connected to (do note that in the routing window, I'd recommend not using the Presets selections with 4th Gen devices and also, the Analogue 1/2 Hardware Inputs should be sent via the DSP). This routing setup is the most generically useful 4th Gen 4i4 configuration:

![image](https://github.com/geoffreybennett/alsa-scarlett-gui/assets/1866657/e8b0608b-e7c6-49da-b5d3-c48771cf10ab)

Please see [USAGE.md — Routing](https://github.com/geoffreybennett/alsa-scarlett-gui/blob/master/USAGE.md#routing) for the 2nd/3rd Gen instructions which are still mostly-applicable.

Here's an example useful mixer configuration:

![image](https://github.com/geoffreybennett/alsa-scarlett-gui/assets/1866657/ffd39c8b-f638-4edb-811e-6bddde9dcb66)

Regards,
Geoffrey.

**Reply by bulkware** [reply]
@geoffreybennett I got this to work yesterday using your instructions. Thank you, this was very helpful and I hope this helps other people as well.
